tests/SBRPTests/PackageData.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+
+namespace SBRPTests;
+
+public class PackageData
+{
+    public static IEnumerable<object[]> Data => new List<object[]>
+    {
+        new object[] { "Microsoft.CodeAnalysis.CSharp.Workspaces,4.5.0", PackageType.Reference },
```

Comment:
What are the reasons for running these 3 ref packages?  If there are specific scenarios it would be useful to call them out.  If there aren't I would be curious why we 3 different packages are run versus 1.

Reply:
These packages are just examples which I picked ramdomly. We could discuss which packages should be covered. Maybe according to the priority of each package.